overlord/snapshotstate/backend/backend.go

```diff
@@ -544,13 +568,72 @@ func (t *importTransaction) Cancel() error {
 
 // Commit will commit a given transaction
 func (t *importTransaction) Commit() error {
-	if err := os.Remove(importInProgressFilepath(t.id)); err != nil {
+	t.committed = true
```

Comment:
Probably not very important, but should commited be set to true only if unlock() didn't return an error?

Reply:
Thanks, I think this makes sense